wgpu/src/backend/web.rs

```diff
@@ -144,11 +144,15 @@ impl crate::ComputePassInner<Context> for ComputePass {
             .dispatch_workgroups_indirect_with_f64(&indirect_buffer.0, indirect_offset as f64);
     }
 
-    fn write_timestamp(&mut self, _query_set: &(), _query_index: u32) {
-        panic!("WRITE_TIMESTAMP_INSIDE_PASSES feature must be enabled to call write_timestamp in a compute pass")
```

Comment:
I think we should keep the wording about the lacking feature as the reason for failure.

Reply:
There should be no need to change feature detection, as it has already been done in wgpu-hal. `WRITE_TIMESTAMP_INSIDE_PASSES` is a native-only feature, web-sys does not implement `write_timestamp` for compute and render pass.

Reply:
I agree, but the story we should be presenting to the user is that WebGPU is just another device, except this device never supports `WRITE_TIMESTAMP_INSIDE_PASSES` so the error itself is not unique to wasm, just a property of the feature set of the device.